scripts/bloom-inference-server/examples/server_request.py

```diff
@@ -0,0 +1,37 @@
+import requests
+
+
+def main():
+    url = "http://127.0.0.1:5000/generate/"
+
+    request_body = {
+        "text": "DeepSpeed is a machine learning framework",
+        "max_new_tokens": 40
+    }
+    response = requests.post(
+        url=url,
+        json=request_body,
+        verify=False
+    )
+    print(response.json())
```

Comment:
```suggestion
    print(response.json(), "\n")
```
let's make the output a little bit more user-friendly out of the box.